docs/usage.md

```diff
@@ -0,0 +1,111 @@
+| [Home](../README.md) |
+|----------------------|
+
+# Usage
+
+On the IOC search page, enter indicators separated by a line break. Maximum of 10 indicators are allowed. Click the button <img src="./res/icon-ioc-search.svg" alt="IOC search icon"> **Search** when done.
```

Comment:
Maximum of 10 indicators are allowed --> imo, this sentence can go as a notes

Reply:
Done